<details><summary>Linter check found the following problems:</summary>
Traceback (most recent call last):
  File "/opt/conda/envs/linter/bin/conda-lint", line 11, in <module>
    sys.exit(main())
             ^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/anaconda_linter/run.py", line 107, in main
    result = linter.lint(
             ^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/anaconda_linter/lint/__init__.py", line 620, in lint
    msgs = self.lint_file(
           ^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/anaconda_linter/lint/__init__.py", line 692, in lint_file
    messages = self.lint_recipe(
               ^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/anaconda_linter/lint/__init__.py", line 748, in lint_recipe
    res = self.check_instances[check].run(recipe, fix)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/anaconda_linter/lint/__init__.py", line 247, in run
    self.check_recipe(recipe)
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/anaconda_linter/lint/check_build_help.py", line 555, in check_recipe
    o = int(path.split("/")[1])
        ^^^^^^^^^^^^^^^^^^^^^^^
ValueError: invalid literal for int() with base 10: 'host'

ERROR conda.cli.main_run:execute(47): `conda run conda-lint /tmp/abs_7d7l6ma7ly/clone` failed. (See above for error)
</details>

--- a/recipe/_update_spacy_recipe.py
+++ b/recipe/_update_spacy_recipe.py
@@ -13,8 +13,9 @@ BUILD_NUMBER = "0"
 
 # see https://github.com/conda-forge/spacy-models-feedstock/issues/2
 SKIP_PATTERNS = [
+    # Example (keep this for the future)
     # needs sudachipy https://github.com/conda-forge/staged-recipes/issues/18871
-    "ja_core*",
+    # "ja_core*",
 ]
 SKIP_PIP_CHECK = {
     # Example (keep this for the future)

--- a/recipe/conda_build_config.yaml
+++ b/recipe/conda_build_config.yaml
@@ -13,6 +13,7 @@ spacy_lang:
   - fr
   - hr
   - it
+  - ja
   - ko
   - lt
   - mk

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -179,6 +179,24 @@ source:
     folder: it_core_news_sm
 {% endif %}  # it
 
+{% if spacy_lang == "ja" %}
+  - url: https://github.com/explosion/spacy-models/releases/download/ja_core_news_lg-{{ version }}/ja_core_news_lg-{{ version }}.tar.gz
+    sha256: 4d4f58147ca570a1f12882f1c5266ae3eafdda5df3e2e8fd4875bfd19a2163ed
+    folder: ja_core_news_lg
+
+  - url: https://github.com/explosion/spacy-models/releases/download/ja_core_news_md-{{ version }}/ja_core_news_md-{{ version }}.tar.gz
+    sha256: dd0f49e126e65dea9d36e029cd1590e87cc692dd013c872e5768df5b0e3e1754
+    folder: ja_core_news_md
+
+  - url: https://github.com/explosion/spacy-models/releases/download/ja_core_news_sm-{{ version }}/ja_core_news_sm-{{ version }}.tar.gz
+    sha256: 28f71d9855b36c02568643ca32675d74541dca0263f24553bd50901feaeec9a9
+    folder: ja_core_news_sm
+
+  - url: https://github.com/explosion/spacy-models/releases/download/ja_core_news_trf-{{ version }}/ja_core_news_trf-{{ version }}.tar.gz
+    sha256: 3982ba7b47134f4da03da0f615b270a2e950311cc9f2d4cb436ab91c7fd13776
+    folder: ja_core_news_trf
+{% endif %}  # ja
+
 {% if spacy_lang == "ko" %}
   - url: https://github.com/explosion/spacy-models/releases/download/ko_core_news_lg-{{ version }}/ko_core_news_lg-{{ version }}.tar.gz
     sha256: 49a92fcba4320cce8140eee50cbc597e742d5393a8b252889bd69310162761e0
@@ -1529,6 +1547,141 @@ outputs:
         - it_core_news_sm/LICENSE
         - it_core_news_sm/LICENSES_SOURCES
 {% endif %}  # it
+
+{% if spacy_lang == "ja" %}
+  - name: spacy-model-ja_core_news_lg
+    build:
+      noarch: python
+      script:
+        - cd ja_core_news_lg && {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+    requirements:
+      host:
+        - python
+        - pip
+        - setuptools
+        - wheel
+      run:
+        - python >=3.6
+        - spacy {{ spacy }}
+        - sudachidict-core >=20211220
+        - sudachipy >=0.5.2,!=0.6.1
+    test:
+      imports:
+        - ja_core_news_lg
+      requires:
+        - pip
+      commands:
+        - pip check
+        - python -c "__import__('ja_core_news_lg').load()"
+        - python -c "__import__('spacy').load('ja_core_news_lg')"
+    about:
+      summary: Japanese pipeline optimized for CPU.
+      description: |-
+        Components: tok2vec, morphologizer, parser, senter, ner, attribute_ruler.
+      license_file:
+        - ja_core_news_lg/LICENSE
+        - ja_core_news_lg/LICENSES_SOURCES
+
+  - name: spacy-model-ja_core_news_md
+    build:
+      noarch: python
+      script:
+        - cd ja_core_news_md && {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+    requirements:
+      host:
+        - python
+        - pip
+        - setuptools
+        - wheel
+      run:
+        - python >=3.6
+        - spacy {{ spacy }}
+        - sudachidict-core >=20211220
+        - sudachipy >=0.5.2,!=0.6.1
+    test:
+      imports:
+        - ja_core_news_md
+      requires:
+        - pip
+      commands:
+        - pip check
+        - python -c "__import__('ja_core_news_md').load()"
+        - python -c "__import__('spacy').load('ja_core_news_md')"
+    about:
+      summary: Japanese pipeline optimized for CPU.
+      description: |-
+        Components: tok2vec, morphologizer, parser, senter, ner, attribute_ruler.
+      license_file:
+        - ja_core_news_md/LICENSE
+        - ja_core_news_md/LICENSES_SOURCES
+
+  - name: spacy-model-ja_core_news_sm
+    build:
+      noarch: python
+      script:
+        - cd ja_core_news_sm && {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+    requirements:
+      host:
+        - python
+        - pip
+        - setuptools
+        - wheel
+      run:
+        - python >=3.6
+        - spacy {{ spacy }}
+        - sudachidict-core >=20211220
+        - sudachipy >=0.5.2,!=0.6.1
+    test:
+      imports:
+        - ja_core_news_sm
+      requires:
+        - pip
+      commands:
+        - pip check
+        - python -c "__import__('ja_core_news_sm').load()"
+        - python -c "__import__('spacy').load('ja_core_news_sm')"
+    about:
+      summary: Japanese pipeline optimized for CPU.
+      description: |-
+        Components: tok2vec, morphologizer, parser, senter, ner, attribute_ruler.
+      license_file:
+        - ja_core_news_sm/LICENSE
+        - ja_core_news_sm/LICENSES_SOURCES
+
+  - name: spacy-model-ja_core_news_trf
+    build:
+      noarch: python
+      script:
+        - cd ja_core_news_trf && {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+    requirements:
+      host:
+        - python
+        - pip
+        - setuptools
+        - wheel
+      run:
+        - python >=3.6
+        - spacy {{ spacy }}
+        - spacy-transformers >=1.2.0.dev0,<1.3.0
+        - sudachidict-core >=20211220
+        - sudachipy >=0.5.2,!=0.6.1
+    test:
+      imports:
+        - ja_core_news_trf
+      requires:
+        - pip
+      commands:
+        - pip check
+        - python -c "__import__('ja_core_news_trf').load()"
+        - python -c "__import__('spacy').load('ja_core_news_trf')"
+    about:
+      summary: Japanese transformer pipeline (cl-tohoku/bert-base-japanese-char-v2).
+      description: |-
+        Components: transformer, morphologizer, parser, ner.
+      license_file:
+        - ja_core_news_trf/LICENSE
+        - ja_core_news_trf/LICENSES_SOURCES
+{% endif %}  # ja
 
 {% if spacy_lang == "ko" %}
   - name: spacy-model-ko_core_news_lg

